Client/Assets/ContextMenu.js

```diff
@@ -74,6 +98,9 @@ addEventListener("touchstart", function (event) {
   var data = {};
   var element = event.target;
 
+  element.addEventListener("touchend", handleTouchEnd, true);
```

Comment:
I think we can remove the `true` here (for using capture) since we're now attaching the listener directly to the element.